twitter.py

```diff
@@ -0,0 +1,87 @@
+"""
+This file gets the mentions on twitter and creates a spotify playlist then retweets it
+back to the user who requested it.
+"""
+import os
+import re
+import tweepy
+from createSpotifyPlaylist import create_spotify_playlist_from_search
+
+auth = tweepy.OAuthHandler(os.environ['consumer_key'], os.environ['consumer_secret_key'])
+auth.set_access_token(os.environ['access_token'], os.environ['access_token_secret'])
+api = tweepy.API(auth)
+
+
+def get_search_query(tweet:str):
+    """Functio to get search query from tweet
```

Comment:
typo

Reply:
fixed the typo in this commit [here](https://github.com/GaelGil/apiPractice/commit/24d5855d83265e21000dd2b174bee22fdb6850ea)